code/game/antagonist/antagonist.dm

```diff
@@ -112,7 +112,7 @@
 			log_debug("[key_name(player)] is not eligible to become a [role_text]: They are blacklisted for this role!")
 		else if(player_is_antag(player))
 			log_debug("[key_name(player)] is not eligible to become a [role_text]: They are already an antagonist!")
-		else if(establish_db_connection(dbcon) && required_age && required_age > player.current.client?.player_age)
+		else if(dbcon?.IsConnected() && required_age && required_age > player.current.client?.player_age)
```

Comment:
Please don't change this. The logic may not be equivalent if establish_db_connection gets changed.